shell/edit/provisioning.cattle.io.cluster/rke2.vue

```diff
@@ -1926,16 +2020,40 @@ export default {
           <h3>
             {{ t('cluster.rke2.security.header') }}
           </h3>
-          <div class="row">
+
+          <div
+            v-if="psaOptions && needsPSA"
```

Comment:
It looks like `psaOptions` is always defined so I think this could just be `v-if="needsPSA"` (nitpicky of me though)

Should this dropdown be shown when the current user can't load PSAs? Standard users can provision clusters, but can't see PSAs, so they just get a dropdown with nothing but `(None)`. Maybe they should be allowed to add a user-defined value here? Idk

Reply:
@kwwii was mentioning that we should keep `RKE Default`, although it's unclear when we have both the dropdowns if that's fine.

Reply:
Removed extra condition.